_posts/2019-03-18-speed-up-your-pipeline-using-cache.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: post
+title:  Speed up your pipeline using the cache
+date: 2019-03-29
+tags: codebuild aws docker cache continuous-integration
+author: JK Gunnink
+---
+
+Fast feedback in development is an important consideration when deploying software often as
```

Comment:
`fast feedback is important` - repeated twice.  You're making the same point - combine the two scentences

Reply:
But a nice lead in - answers the key question "Why should I read this post?"

Reply:
So this is using Docker cache, rather than codebuild cache?  Needs to be explicit IMO - I was expecting more of a post about caching in codebuild specifically.